modules/gdscript/tests/scripts/highlighter/type_hints.out

```diff
@@ -0,0 +1,32 @@
+0001 class_name MyClass
+>>>> Kw        SN
+0002
+>>>>
+0003 class InnerClass extends RefCounted: pass
+>>>> Kw   SN         SKw     SEt        S Cf
```

Comment:
Yyyyyeah, we need to discuss this somewhere. The arbitrary character count for each identifier is very prone to confusion and hard to decipher. At the same time, so would be using a single character.